### PR TITLE
[ZH] Improve validation

### PR DESCRIPTION
**Pull request by xueron**
Improve validation translation.

### CODE REVIEW

**Comment by sergeyklay** [comment]
@dreamsxin Could you please take a look

File: zh/reference/validation.rst [hunk 1]
```diff
@@ -420,9 +420,9 @@ Phalcon的验证组件中内置了一些验证器：
         }
     }
 
-Avoid validate empty values
+避免验证空值（Avoid validate empty values）
 ---------------------------
```
Comment (dreamsxin) [inline zh/reference/validation.rst]: 横线需要加长到跟标题一样，不然生成会有警告。

**Comment by dreamsxin** [comment]
@sergeyklay Correct content, But `---------------------------` too short.

File: zh/reference/validation.rst (hunk 1 above)
Comment (xueron) [inline zh/reference/validation.rst]: Got it. Not so 熟练 with markdown, sorry. ~-~

**Comment by sergeyklay** [comment]
@xueron This PR cannot be merged due to conflicts. Can you rebase?

**Comment by xueron** [comment]
@sergeyklay OK, I'll fix it.

**Pull request merged**